packages/ui-toolkit/stories/ToggleSwitch.stories.tsx

```diff
@@ -12,27 +12,31 @@ export default {
   }
 };
 
+
 const Template: Story<ToggleSwitchProps> = (args) => {
-  const [checked, setChecked] = React.useState(false);
+  const [ checked, setChecked ] = React.useState(true);
```

Comment:
Its a general practice to use variables with prefix "is" when its a boolean.
In this case please use [ isChecked, setIsChecked ]

Reply:
ohh, got it.